script.js

```diff
@@ -1,5 +1,202 @@
+/* 
+There are 2 players and players take turns.
+When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+After both players have rolled and chosen dice order, the player with the higher combined number wins.
+*/
+
+// Global Var
+player1diceResults = []
+player2diceResults = []
+player1 = ""
+player2 = ""
+player1Number = 1
+player2Number = 2
+// 
```

Comment:
Nice and clear global variables, do remember to declare them with `var`. There are some side effects of not declaring them with `var` which I won't get into, which is why it is recommended to declare it with `var` even if the code still works the way you have it.

Reply:
However, i see that `player1Number` and `player2Number` are subsequently not used anywhere in your code, seems like a good place to use them is e.g. in lines 160, 167, etc where you pass in 1 or 2 to a function